gtest/src/log.rs

```diff
@@ -396,6 +396,40 @@ impl RunResult {
             .flat_map(DecodedCoreLog::try_from_log)
             .collect()
     }
+
+    /// Trying to get the panic log.
+    fn panic_log(&self) -> Option<&CoreLog> {
```

Comment:
I would move that below, under the `panicked` and `assert_panicked_with` to make code more readable (top to bottom read style)